dashboard/db/migrate/20170131165950_add_uniqueness_to_user_proficiency_user_id.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniquenessToUserProficiencyUserId < ActiveRecord::Migration[5.0]
+  def up
```

Comment:
Any recommendations on how to make this reversible? Note that `remove_index` is only reversible with a column parameter. But it does not take a column parameter with a name parameter.